scripts/beesd@.service.in

```diff
@@ -20,7 +20,39 @@ Restart=on-abnormal
 StartupCPUWeight=25
 StartupIOWeight=25
 
-PrivateMounts=true
```

Comment:
Removing private mounts probably introduces an unexpected behavior: The mount point for bees may not be automatically unmounted when the process dies.

Reply:
According to [systemd official doc for PrivateMounts](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#PrivateMounts):

> Other file system namespace unit settings — PrivateMounts=, PrivateTmp=, PrivateDevices=, ProtectSystem=, ProtectHome=, ReadOnlyPaths=, InaccessiblePaths=, ReadWritePaths=, … — also enable file system namespacing in a fashion equivalent to this option. Hence it is primarily useful to explicitly request this behaviour if none of the other settings are used.

And, the fact that I cannot access the mount point from /run/beesd means that the process is already running in a separate mount namespace.

Linux’s namespace (except for the init one) lives only if there’s process in it.
Since beesd is the only process in it, once it exits, the namespace is torn down and the mountpoint in their is also torn down.